src/utils/ddb/ddb_vos.c

```diff
@@ -50,6 +49,35 @@ dv_pool_open(char *path, daos_handle_t *poh)
 	return rc;
 }
 
+int
+dv_pool_destroy(const char *path)
```

Comment:
Would you please to add some test cases for the pool destroy via ddb? On the other hand, since the DDB tool can be used for recovery from disaster, if SMD information does not match pool case or say SMD is corrupted, can we still operate on the pool by force?